Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/UserInterface.java

```diff
@@ -0,0 +1,24 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+public interface UserInterface {
```

Comment:
I think using an interface is great because it makes for easily swappable user interfaces, but see [https://github.com/appetrosyan/JavaCodeReview/pull/1/files#r49899543](my comment) above.